src/server.coffee

```diff
@@ -262,6 +266,91 @@ startAuditUDPServer = (auditUDPPort, bindAddress) ->
 
   return defer
 
+
+# function to start the TLS Audit server
+startAuditTlsServer = (auditTlsPort, bindAddress) ->
```

Comment:
I think you could combine the startAuditTlsServer and the startAuditTcpServer functions here. Rather just have a parameter for tls or not. There is a fair bit of code duplications between these two.